src/WizardsRest/ObjectReader/DoctrineAnnotationReader.php

```diff
@@ -114,12 +114,13 @@ public function getExposedProperties($resource, array $filter)
 
     public function getPropertyValue($resource, string $name)
     {
+        $getterMethod = 'get'.ucfirst($name);
```

Comment:
you have to use getPropertyGetter to keep the @getter feature
you might want to use lcfirst instead of strtolower tho